src/test/ui/consts/rfc-2203-const-array-repeat-exprs/fn-call-in-const.rs

```diff
@@ -1,7 +1,4 @@
-// run-pass
```

Comment:
I think we should just delete this entire folder (or even better, rename it, and use `const { }` blocks everywhere, which should make all the run-pass tests work again and all the other test emit the same diagnostic that they were emitting with the feature.)

Reply:
What's the feature for const-blocks? I can change it but I haven't been up to date with it

Reply:
`inline_const` is the feature name

Reply:
Looking at it more closely, I think is the only file that is affected, but I've moved all of these into a folder under consts.